clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java

```diff
@@ -141,4 +143,8 @@
     @LogMessage(level = INFO)
     @Message(id = 32, value = "Getting remote cache named '%s'. If it does not exist a new cache will be created from configuration template named '%s'; null value uses default cache configuration on the Infinispan Server.")
     void remoteCacheCreated(String remoteCacheName, String cacheConfiguration);
+
+    @LogMessage(level = WARN)
+    @Message(id = 33, value = "Attribute '%s' configured to use deprecated value: %s; use one of the following values instead: %s")
+    void enumValueDeprecated(String attributeName, Object attributeValue, Set<?> supportedValues);
```

Comment:
This is kind of a nit, but this method name kind of encourages misuse, i.e. reuse in a very different context, where any kbase article / google search result looking for the message id would find content that's not meaningful.

Also

s/configured/is configured/g
s/use deprecated/use a deprecated/g

Reply:
will fix